### PR TITLE
(greek_conversion) EpiDoc and CTS conversion work 

### PR DESCRIPTION
**Pull request by lcerrato**
tlg0545 batch #1399

### CODE REVIEW

File: data/tlg0545/tlg002/tlg0545.tlg002.perseus-grc2.xml
Comment (AlisonBabeu) [inline data/tlg0545/tlg002/tlg0545.tlg002.perseus-grc2.xml:1]: hi @lcerrato only thing I would change on this file is in the structured bibliography. 
`<title>Claudii Aeliani de natura animalium libri xvii, varia historia, epistolae, fragmenta, Vol 2</title>`
I would delete Vol 2 here and add it in using the biblScope element! 

The same is true for tlg0545.tlg002.perseus-grc3 below! :)

Comment (lcerrato) [reply data/tlg0545/tlg002/tlg0545.tlg002.perseus-grc2.xml:1]: There is a biblScope element, no?

File: data/tlg0545/tlg003/tlg0545.tlg003.perseus-grc2.xml
[the remark this answers is not in the document]
Comment (lcerrato) [reply data/tlg0545/tlg003/tlg0545.tlg003.perseus-grc2.xml:1]: This file doesn't appear to be completely done/

**Comment by lcerrato** [comment]
@AlisonBabeu I appreciate your looking at these: the headers were/are very confusing!

**Comment by lcerrato** [comment]
@AlisonBabeu Would you mind looking over the metadata here? I don't want to push more files/changes to this batch.

File: data/tlg0545/tlg003/__cts__.xml
Comment (AlisonBabeu) [inline data/tlg0545/tlg003/__cts__.xml:1]: This is missing the volume number.

File: data/tlg0555/tlg004/tlg0555.tlg004.perseus-grc1.xml
Comment (AlisonBabeu) [inline data/tlg0555/tlg004/tlg0555.tlg004.perseus-grc1.xml:1]: So @lcerrato should this be tlg0555.tlg004.perseus-grc2? And also there is no cts_.xml file for the 004 directory.

File: data/tlg0545/tlg003/__cts__.xml
Comment (lcerrato) [reply data/tlg0545/tlg003/__cts__.xml:1]: Your earlier comment said to delete the volumes in the CTS files and keep them in the biblScope? So I deleted them in the CTS files.

File: data/tlg0555/tlg004/tlg0555.tlg004.perseus-grc1.xml
Comment (lcerrato) [reply data/tlg0555/tlg004/tlg0555.tlg004.perseus-grc1.xml:1]: I removed an extra period in the file name.

File: data/tlg0545/tlg003/__cts__.xml
Comment (lcerrato) [reply data/tlg0545/tlg003/__cts__.xml:1]: @AlisonBabeu I think I see the issue: I thought you wanted it removed here but you were talking about the file header errors. I had it in this batch and will re-add now.

**Review by AlisonBabeu — CHANGES_REQUESTED** [review]
I made a comment on one file that needs a minor metadata change and there also appears to be a missing cts_.xml file for tlg0555.tlg004.

**Comment by lcerrato** [comment]
@AlisonBabeu Thank you. tlg0555.tlg004 is not being converted as part of this PR.

File: data/tlg0545/tlg003/__cts__.xml
Comment (AlisonBabeu) [reply data/tlg0545/tlg003/__cts__.xml:1]: @lcerrato Sorry if I was unclear,I usually include it in both places.

**Pull request merged**